bittensor/utils/wallet_utils.py

```diff
@@ -102,3 +102,61 @@ def is_valid_bittensor_address_or_public_key(address: Union[str, bytes]) -> bool
     else:
         # Invalid address type
         return False
+
+
+
+def create_identity_dict(
+    display: str = "",
+    legal: str = "",
+    web: str = "",
+    riot: str = "",
+    email: str = "",
+    pgp_fingerprint: Optional[str] = None,
+    image: str = "",
+    info: str = "",
+    twitter: str = "",
+) -> dict:
+    """
+    Creates a dictionary with structure for identity extrinsic.
```

Comment:
Do we have the requirements for each parameter? Might be useful to document here if known.

Reply:
Nothing specific, but there is a max of 64bytes for each param. So there is a limit there